build.gradle

```diff
@@ -536,7 +532,7 @@ task testIntegration(type: Test) {
             maxRetries = 2
         }
     }
-    maxParallelForks = 2
+    maxParallelForks = 8
```

Comment:
I had been tinkering with this locally as well.  I was reading primers [here](https://docs.gradle.org/current/userguide/performance.html) and was using the suggested "maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1". It will be interesting to see how this does with the new Mac M processors.

Reply:
Right, exactly.  It's a good point that we should maybe make this one based of availableProcessors().  It's kinda tough since connected and unit and integration all have different runtime profiles (e.g. how many parallel runs can be supported)